set allow_duplicate_ip to false (#2992)
tendermint/tendermint: docs/tendermint-core/configuration.md

prometheus_listen_addr = ":26660"

# Maximum number of simultaneous connections.

> # If you want to accept more significant number than the default, make sure

Reviewer: ?
```suggestion
# If you want to accept a larger number than the default, make sure
```
